supergloo/docs/mesh/tutorials-2-root-cert.md

```diff
@@ -0,0 +1,220 @@
+kind: Secret
+metadata:
+  annotations:
+    resource_kind: '*v1.TlsSecret'
+  creationTimestamp: 2019-03-19T19:59:49Z
+  name: my-root-ca
+  namespace: supergloo-system
+  resourceVersion: "451018"
+  selfLink: /api/v1/namespaces/supergloo-system/secrets/my-root-ca
+  uid: 8d4bbd3d-4a81-11e9-9c8c-3c78d8017c4a
+type: Opaque
+```
+
+Finally, we must tell SuperGloo to use this secret for certificate provisioning in our mesh: 
+
+```bash
+supergloo set rootcert --target-mesh supergloo-system.istio \  
+    --tls-secret supergloo-system.my-root-ca
+
++----------+-------+------+
+|   MESH   | TYPE  | MTLS |
++----------+-------+------+
+| my-istio | Istio | true |
++----------+-------+------+
+```
+
+> Note: `target-mesh` should be set to the `NAMESPACE.NAME` of the managed mesh you'd like to configure. To
+> view the list of available meshes, run `kubectl get mesh.supergloo.solo.io --all-namespaces`
+ 
+
+Once that's done, we'll use some `kubectl` commands to inspect the certificates Istio has distributed to the 
+Bookinfo app. 
+
+Let's download the root certificate was loaded to the sidecar for one of the bookinfo pods: 
+
+```bash
+RATINGSPOD=`kubectl get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'`
+kubectl exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/root-cert.pem > pod-root-cert.pem
+```
+
+Using `diff`, we should see that the `pod-root-cert.pem` matches our own `root-cert.pem`:
```

Comment:
I ran through these examples and found they're not the same. Can you paste the output you're expecting? Can you give it a quick run through and see what you get?

Reply:
takes some time, i added a note about that